src/BenchmarksApps/TechEmpower/Minimal/Templates/Fortunes.cshtml

```diff
@@ -0,0 +1,2 @@
+﻿@inherits RazorSliceHttpResult<List<Fortune>>
+<!DOCTYPE html><html><head><title>Fortunes</title></head><body><table><tr><th>id</th><th>message</th></tr>@foreach (var item in Model){<tr><td>@item.Id</td><td>@item.Message</td></tr>}</table></body></html>
```

Comment:
How much slower is that without spaces ;)

Reply:
You mean with spaces? It's more to transmit so slower. This is an optimal template.

Reply:
> This is an optimal template.

That was sarcasm... did you measure the difference?

Reply:
Yah it's measurable and we're trying to ensure the switch to Razor loses as little ground as possible. Do you have a genuine issue with it?